### PR TITLE
Added functionality to update the side summary panel on new character

### PR DESCRIPTION
**Pull request by jango2106**
I updated the summary panel in New Character to pull some data from the character object:
-Name
-Class
-Race
-Attributes and Mods

image

### CODE REVIEW

File: dnd-project/Core/BusinessModels/Character.cs [hunk 1]
```diff
@@ -340,6 +340,32 @@ public void GetObjectData(SerializationInfo info, StreamingContext context)
...
+            return attributesList;
+        }
+        
+        /// <summary>
+        /// Gets name of the current race
+        /// </summary>
+        /// <returns>Name of the race or "None" if race is null</returns>
+        public string GetRace()
+        {
+            return characterRace == null ? "None" : characterRace.Name;
```
Comment (ahoopes16) [inline dnd-project/Core/BusinessModels/Character.cs:358]: Mmmmm, ternary

File: dnd-project/Core/ViewModels/CharacterValues.cs [hunk 2]
```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
```
Comment (ahoopes16) [inline dnd-project/Core/ViewModels/CharacterValues.cs:6]: Wow that got rid of a whole lot of imports. Were we needlessly bringing them in?

**Review by ahoopes16 — APPROVED** [review]
Looks good to me! As much as I can remember this code base at least lol

File: dnd-project/Core/ViewModels/CharacterValues.cs (hunk 2 above)
Comment (jango2106) [reply dnd-project/Core/ViewModels/CharacterValues.cs:6]: I think they are automatically imported on new classes

File: dnd-project/Core/BusinessModels/Character.cs (hunk 1 above)
Comment (jango2106) [reply dnd-project/Core/BusinessModels/Character.cs:358]: Very tasty

**Pull request merged**